src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/isconnected.md

```diff
@@ -16,7 +16,7 @@ Indicates whether the [CDP](https://chromedevtools.github.io/devtools-protocol/)
 
 | Type    | Description                                                                                    |
 | ------- | ---------------------------------------------------------------------------------------------- |
-| boolean | Returns `true` if [Browser](/javascript-api/k6-experimental/browser/browser-class/) is connected to the browser application. Otherwise, returns `false`. |
+| boolean | Returns `true` if [the browser module](/javascript-api/k6-experimental/browser) is connected to the browser application. Otherwise, returns `false`. |
```

Comment:
```suggestion
| boolean | Returns `true` if the [browser module](/javascript-api/k6-experimental/browser) is connected to the browser application. Otherwise, returns `false`. |
```